lightstep/resource_user_role_binding.go

```diff
@@ -0,0 +1,201 @@
+package lightstep
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/lightstep/terraform-provider-lightstep/client"
+)
+
+func resourceUserRoleBinding() *schema.Resource {
+	return &schema.Resource{
+		Description:   "This resources is under development and is not generally available yet.",
+		CreateContext: resourceUserRoleBindingCreateOrUpdate,
+		ReadContext:   resourceUserRoleBindingRead,
+		UpdateContext: resourceUserRoleBindingCreateOrUpdate,
+		DeleteContext: resourceUserRoleBindingDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceUserRoleBindingImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"role": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true, // changing role or project requires a new tf resource to ensure permissions are properly removed.
+				Description: "Role's name being granted with this role binding.",
+				ValidateFunc: validation.StringInSlice([]string{
+					"Organization Admin",
+					"Organization Editor",
+					"Organization Viewer",
+					"Organization Restricted Member",
+					"Project Editor",
+					"Project Viewer",
+				}, false),
+			},
+			"project": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true, // changing role or project requires a new tf resource to ensure permissions are properly removed.
+				Description: "Name of the project where this project, if omitted the role will be applied to the organization",
```

Comment:
what do you think of this phrasing?
```suggestion
				Description: "Name of the project where this role will be applied; if omitted the role will be applied to the organization",
```